pkgs/applications/misc/hyperspace/default.nix

```
@@ -0,0 +1,61 @@
+{ stdenv, fetchurl, lib
+, autoPatchelfHook, dpkg
+, libX11, libXext, libXi, libXau, libXrender, libXft, libXmu, libSM, libXcomposite, libXfixes, libXpm
+, libXinerama, libXdamage, libICE, libXtst, libXaw, fontconfig, pango, cairo, glib, libxml2, atk, gtk3
+, gdk-pixbuf, nss, nspr, libXScrnSaver, alsa-lib, udev, libdrm, libGL, mesa
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hyperspace";
+  version = "1.1.4";
+
+  src = fetchurl {
+    url = "https://github.com/hyperspacedev/${pname}/releases/download/v${version}/${pname}_${version}_amd64.deb";
```

```suggestion
    url = "https://github.com/hyperspacedev/hyperspace/releases/download/v${version}/hyperspace_${version}_amd64.deb";
```
easy to visit links are important if you want to research with one click if this can be build from source.